docs/pages/tutorial/initialize-app.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Create your first app
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+In this chapter, you'll learn how to create a new Expo project and how to get it running.
+
+## Prerequisites
+
+You'll need the following tools to get started:
+
+- Install [Expo Go](https://expo.dev/client) on your physical device.
+- Prepare your computer by [installing the required tools and dependencies](https://docs.expo.dev/get-started/installation/#requirements).
+
+This tutorial also assumes you have basic knowledge of JavaScript and React or React Native. If you have never written React code, go through [the official React tutorial](https://reactjs.org/tutorial/tutorial.html) first. For more resources, see [additional resources](https://docs.expo.dev/next-steps/additional-resources/).
+
+## Step 1: Initialize a new Expo app
+
+We provide two different ways to get started:
+
+- [Initialize with the starter template](#initialize-with-the-starter-template)
+- [Manually initialize with `create-expo-app`](#alternate-manually-initialize-with--create-expo-app)
+
+Choose a way to initialize your app and open the project repository in your favorite code editor or IDE after initializing it. Throughout this tutorial, we use VS Code for our examples.
+
+### Initialize with the starter template
+
+In the directory of your choice with your preferred terminal, download the starter project using an Expo template:
+
+<Terminal cmd={['$ npx create-expo-app -t expo-tutorial-template']} />
```

Comment:
Is the starter template referred to here in a separate PR?

Reply:
Yeah in this PR: https://github.com/expo/expo/pull/18885#issuecomment-1230195289